src/stories/Library/pagefold/Pagefold.tsx

```diff
@@ -36,10 +38,30 @@ export const Pagefold: React.FC<PagefoldProps> = (props) => {
     ),
   };
 
-  return (
+  if (button) {
+    return (
+      <button
+        type="button"
+        aria-pressed={status === "selected"}
+        className={classes.wrapper}
+      >
+        <div className={classes.triangle} />
+        {children}
+      </button>
+    );
+  }
+
```

Comment:
I think it is a confusing to control the type of pageFold from props especially when you also introduce the `compProps`.
In this case I would probably go the imperative way and simply create a component for each purpose. Eg.: `PageFoldButton`. And then share the logic about getting the right classes via clsx.

Btw discovered that the `baseClass`const should be used [here](https://github.com/reload/dpl-design-system/blob/e15fd7a4d829695269ff4784d1d5abee127ca9c7/src/stories/Library/pagefold/Pagefold.tsx#L36) instead of defining it again.